src/shared.rs

```diff
@@ -3,16 +3,22 @@
 use std::time::Duration;
 
 use anyhow::{Context, Result};
+use futures::{SinkExt, StreamExt};
 use serde::de::DeserializeOwned;
 use serde::{Deserialize, Serialize};
-use tokio::io::{self, AsyncBufRead, AsyncBufReadExt, AsyncRead, AsyncWrite, AsyncWriteExt};
+use tokio::io::{self, AsyncRead, AsyncWrite};
+
 use tokio::time::timeout;
+use tokio_util::codec::{AnyDelimiterCodec, Framed};
 use tracing::trace;
 use uuid::Uuid;
 
 /// TCP port used for control connections with the server.
 pub const CONTROL_PORT: u16 = 7835;
 
+/// TCP max message length server.
+pub const MAX_FRAME_LENGTH: usize = 200;
```

Comment:
Could be better written, like "Maximum byte length for a JSON message over TCP."